packages/stencil/components/form-field/stencil.config.ts

```diff
@@ -0,0 +1,29 @@
+import { Config } from '@stencil/core';
+import { getStencilConfig } from '@ovhcloud/ods-stencil/libraries/stencil-core';
+import * as jestConfig from './jest.config';
+
+const args = process.argv.slice(2);
+
+export const config: Config = getStencilConfig({
+  namespace: 'osds-form-field',
+  args,
+  jestConfig: jestConfig.default,
+  reactOutput: {
+    componentCorePackage: '@ovhcloud/ods-stencil/components/form-field',
+    // exclude peer dependencies that corresponds to www usage
+    excludeComponents: []
```

Comment:
you may need to exclude components defined on your global files (`text`, `input`, ...)